ui/components/app/add-network/add-network.js

```diff
@@ -147,24 +149,23 @@ const AddNetwork = () => {
             </Box>
           )}
           <Box
-            margin={
-              getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
-                ? [0, 0, 1, 0]
-                : [4, 0, 1, 0]
-            }
+            marginTop={getEnvironmentType() === ENVIRONMENT_TYPE_POPUP ? 0 : 4}
+            marginBottom={1}
             className="add-network__main-container"
           >
             <Typography
               variant={TYPOGRAPHY.H6}
               color={COLORS.TEXT_ALTERNATIVE}
-              margin={[4, 0, 0, 0]}
+              margin={0}
+              marginTop={4}
             >
               {t('addFromAListOfPopularNetworks')}
             </Typography>
             <Typography
               variant={TYPOGRAPHY.H7}
               color={COLORS.TEXT_MUTED}
-              margin={[4, 0, 3, 0]}
+              marginTop={4}
+              marginBottom={3}
             >
               {t('popularCustomNetworks')}
             </Typography>
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props